rule/utils.go

```diff
@@ -140,3 +141,81 @@ func isCallToExitFunction(pkgName, functionName string) bool {
 func newInternalFailureError(e error) []lint.Failure {
 	return []lint.Failure{lint.NewInternalFailure(e.Error())}
 }
+
+// hasUpperCaseLetter checks if string contains at least one upper case letter
+func hasUpperCaseLetter(s string) bool {
+	for _, r := range s {
+		if unicode.IsUpper(r) {
+			return true
+		}
+	}
+	return false
+}
+
+// hasLowerCaseLetter checks if string contains at least one lower case letter
+func hasLowerCaseLetter(s string) bool {
+	for _, r := range s {
+		if unicode.IsLower(r) {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
now you refactored, is this used except in the tests?

Otherwise, it can be removed

 ```suggestion
```

Reply:
@ccoVeille good point, hasLowerCaseLetter removed :P